tests/system/action/motion_comment/test_delete.py

```diff
@@ -13,3 +24,18 @@ def test_delete_wrong_id(self) -> None:
         response = self.request("motion_comment.delete", {"id": 111})
         self.assert_status_code(response, 400)
         self.assert_model_exists("motion_comment/112")
+
+    def test_delete_no_permissions(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion_comment.delete",
+            {"id": 111},
+        )
+
+    def test_delete_permissions(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion_comment.delete",
+            {"id": 111},
+            Permissions.Motion.CAN_SEE,
+        )
```

Comment:
Please also add the 2 additional tests (user not in write groups and user not in write groups, but is admin) for the delete action

Reply:
Add 2 tests. and update the default test.